### ISSUE TITLE
WooCommerce Status dashboard widget missing in 5.5.2

### ISSUE THREAD

**Issue by superbmario** [woocommerce/woocommerce#30347]
Just updated to 5.5.2 and the [WooCommerce Status dashboard widget](https://docs.woocommerce.com/wp-content/uploads/2020/03/dashboard-widgets-2.png?w=950) has gone. It was showing in version 5.5.1. Is this an intentional decision to depreciate the widget?

System report:

### WordPress Environment ###

WC Version: 5.5.2
REST API Version: ✔ 5.5.2
WC Blocks Version: ✔ 5.3.3
Action Scheduler Version: ✔ 3.2.1
WC Admin Version: ❌ 2.4.4
Log Directory Writable: ✔
WP Version: ❌ 5.6.4 - There is a newer version of WordPress available (5.8)
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: –
Language: en_US
External object cache: –

### Server Environment ###

Server Info: LiteSpeed
PHP Version: 7.4.21
PHP Post Max Size: 512 MB
PHP Time Limit: 900
PHP Max Input Vars: 8000
cURL Version: 7.71.0
OpenSSL/1.1.1d

SUHOSIN Installed: –
MySQL Version: 10.3.30-MariaDB-log-cll-lve
Max Upload Size: 32 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ❌ Your server does not have the SoapClient class enabled - some gateway plugins which use SOAP may not work as expected.
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

**Comment by superbmario** [comment]
Looking at https://docs.woocommerce.com/document/dashboard-widgets/ it states:

_Note: When WooCommerce Setup Wizard isn’t completed, WooCommerce Status widget will not appear. Instead, WooCommerce Setup widget will show._

However, I am running a store which has been active for several years. Does this mean I need to re-do the wizard? Is there a way I can skip it as I don't want to change any settings.

**Comment by rodelgc** [comment]
Hi @superbmario,

While our goal is to address all the issues reported in this repository, GitHub should be treated as a place to report confirmed bugs only.

The type of issue you submitted looks like a support request which may or may not reveal a bug once proper troubleshooting is done. In order to confirm the bug, please follow one of the steps below:

- Review [WooCommerce Self-Service Guide](https://docs.woocommerce.com/document/woocommerce-self-service-guide/) to see if the solutions listed there apply to your case;
- If you are a paying customer of WooCommerce, contact WooCommerce support by [opening a ticket or starting a live chat](https://woocommerce.com/contact-us/);
- Make a post on [WooCommerce community forum](https://wordpress.org/support/plugin/woocommerce/)

If you confirm the bug, please provide us with clear steps to reproduce it.

I am closing this issue for now as it seems to be a support request and not a bug. If I missed something, please leave a comment and we will take a second look.

**Comment by superbmario** [comment]
OK thanks @rodelgc. As requested I have now opened a ticket with woocommerce.com support.

FWIW I believe this is a bug when WooCommerce Admin is disabled.

**Comment by superbmario** [comment]
Sorry, I realise this is already closed but I have found more information which may help.

The issue seems to be caused by line 66 of this commit [1cf6b9d39e7a74cb8da748731785f5dae3edec72](https://github.com/woocommerce/woocommerce/commit/1cf6b9d39e7a74cb8da748731785f5dae3edec72#diff-784a2af1fa979a2c43ac849154bd4ce16140900620e47767e36b0b305901f840).

@Konamiman - is this commit intended to disable the WooCommerce Status widget if the site has disabled WC Admin via the 'woocommerce_admin_disabled' filter? If so, that is unfortunate as this was not the case in version 5.5.1. I have now rolled back this file to the [previous version](https://github.com/woocommerce/woocommerce/blob/00a272fa85a724a31eb813218ed8b92268c497fb/includes/admin/class-wc-admin-dashboard.php) and the WooCommerce Status widget appears as normal again. Is there any way we can change it back to that whilst keeping your intended changes at too?

**Comment by jonathansadowski** [comment]
Hi @superbmario,

Thanks for reporting this. It looks like we may have been a bit too aggressive with restricting the entire widget, and probably could have been more selective in disabling just the features that rely on WooCommerce Admin when it is disabled. I'm going to reopen this issue and use it to track a revisit of the PR it was applied (#29613)!

**Comment by pipdig** [comment]
I think the comments at [https://wordpress.org/support/topic/woocommerce-status-meta-box-turns-to-setup-woocommerce/](https://wordpress.org/support/topic/woocommerce-status-meta-box-turns-to-setup-woocommerce/) are related to this issue. I can confirm I'm also seeing the same result after updating WC to version 5.7.1

**Comment by pacmanito** [comment]
I can confirm that the bug persists. Even after proceeding with "onboarding" the status widget is not shown until you enable WooCommerce Admin.

**Comment by strarsis** [comment]
Same issue, there is an empty "WooCommerce Setup" widget on the Dashboard and 
the usual WooCommerce widget with the order, stock and revenue information is missing.
How can I get it back?

Edit: It is a completely different widget - the wiget that is empty is the WooCommerce Setup progress widget.
The widget I want is the WooCommerce status widget.

`$this->should_display_widget()` returns false for the normal WooCommerce status widget!
In that function `get_option( 'woocommerce_task_list_complete' )` and `get_option( 'woocommerce_task_list_hidden' )` are both `no` - and not `yes` as at least one is expected to be.

In WooCommerce there is code to update the option `woocommerce_task_list_hidden` (none for `woocommerce_task_list_complete` by the way). And this code apparently wasn't called correctly and caused this incorrect state. There are no tasks left, but the option `woocommerce_task_list_complete` is used instead - not the actual state of all completed tasks.
The WooCommerce Admin (`woocommerce-admin`) does update the option, too! And I am sure it caused this inconsistent state!

Manually invoking `update_option( 'woocommerce_task_list_hidden', 'yes' );` fixes the issue - but this should be done automatically by WooCommerce itself. Or using the WP CLI to update that function to `yes`:
`wp option update woocommerce_task_list_hidden yes` (I had to run the CLI command twice, first time an error occured, that the option couldn't be updated, strange).

**Comment by rrennick** [comment]
Closing as a duplicate of #30661.

**Comment by KoolPal** [comment]
@rrennick @vedanshujain Few observations:
1. Normally an older issue is retained and the newer duplicate issue(s) is/are merged / closed. This gives a clear indication of aging and hence may help in priority.
2. This older issue #30347 has MORE details / information than the newer duplicate issue #30661 retained by you.

Just two cents. As long as this issue get some resolution after pending for 2 years.

Thanks

**Comment by rrennick** [comment]
> This gives a clear indication of aging and hence may help in priority.

@KoolPal I appreciate the feedback. For this point: in this case both issues are over 2 years old and have a 6 week difference in age.
